controllers/dataplane/openstackdataplanenodeset_controller.go

```diff
@@ -120,6 +120,10 @@ func (r *OpenStackDataPlaneNodeSetReconciler) GetLogger(ctx context.Context) log
 // +kubebuilder:rbac:groups="image.openshift.io",resources=imagetags,verbs=get;list;watch
 // +kubebuilder:rbac:groups="image.openshift.io",resources=imagestreamtags,verbs=get;list;watch
 
+// RBAC for ImageContentSourcePolicy and MachineConfig
+// +kubebuilder:rbac:groups="operator.openshift.io",resources=imagecontentsourcepolicy,verbs=get;list
+// +kubebuilder:rbac:groups="machineconfiguration.openshift.io",resources=machineconfig,verbs=get;list
+
```

Comment:
do we need to watch them for changes? I.e. can the ICSP or the MC change after the initial deployment?

Reply:
True, we probably do need a Watcher for the MachineConfig. Probably not so much for the ImageContentSourcePolicy though. We're only using that to determine whether we need to configure mirrors at all.

Reply:
Last commit on the PR now aims to implement this watcher